uaclient/yaml.py

```diff
@@ -8,29 +8,31 @@
 
 try:
     import yaml
-except ImportError:
-    LOG.error(MISSING_YAML_MODULE.msg)
+except ImportError as e:
+    LOG.exception(e)
     print(MISSING_YAML_MODULE.msg, file=sys.stderr)
     sys.exit(1)
 
 
 def safe_load(stream):
     try:
         return yaml.safe_load(stream)
-    except AttributeError:
-        msg = BROKEN_YAML_MODULE.format(path=yaml.__path__).msg
-        LOG.error(msg)
-        print(msg, file=sys.stderr)
+    except AttributeError as e:
+        LOG.exception(e)
```

Comment:
Should we log here also a similar message to what we are printing ?

Reply:
We can if you want, but `LOG.exception` will include the error and stacktrace pointing to this line, so I think we'll be able to see what happened in the logs